commands/login.js

```diff
@@ -1,49 +1,49 @@
-'use strict';
+const cli = require('heroku-cli-util')
+const child = require('child_process')
+const log = require('../lib/log')
 
-const cli = require('heroku-cli-util');
-const co = require('co');
-const child = require('child_process');
-
-module.exports = function(topic) {
+module.exports = function (topic) {
   return {
     topic: topic,
     command: 'login',
-    flags: [{ name: 'verbose', char: 'v', hasValue: false }],
-    description: 'Logs in to the Heroku Docker registry',
+    flags: [{name: 'verbose', char: 'v', hasValue: false}],
+    description: 'logs in to the Heroku Container Registry',
```

Comment:
so I recently ran through our entire help and audited everything: https://github.com/heroku/cli-engine/issues/196

What I found was there is a mix of using present tense and another type that I think might be "imperative mood" but I'm not sure. Anyways, this "imperative mood" (if that is the correct term) is used much more commonly and I think makes more sense.

So in this case it would be "log in to the Heroku Container Registry". Not "logs in to the Heroku Container Registry". I plan to add this to the style guide.

Also, I'm not sure what language we use elsewhere in documentation, but I'm pretty sure in the CLI we never use "log in" as 2 words, it's always "login" and "logout". So I think my first comment here was wrong for a few reasons.